index.html

```diff
@@ -837,7 +837,7 @@ <h4 class="subtitle chapter-subtitle">Visual language as a device for marketing<
 					<figcaption>Fig.25 Сладка работа (Sweet
 						job) by Tony Dacheva & Orchestra Kristal</figcaption>
 					<figure class="videoWrapper">
-						<iframe src="https://www.youtube.com/embed/ZsBaZH5sHhw" loading="lazy"
+						<iframe src="https://www.youtube.com/embed/X3295YpPN_Y" loading="lazy"
```

Comment:
This version of the video should be of the original 1998 clip. 
So better use this video: `xDqpYQ7tUbc`